hicommonwealth/commonwealth
# Fix new proposal page infinite loading + community not loading at all for slow connections

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/3258

## Description of Changes
1- The layout page had a race condition (kind of). When the chain load takes time, cow-bobber shows up infinitely. This was because whenever the chain init happens, even if in progress the layout was redrawn and it loaded the current page level component, now sometimes on fast connections this worked without issues. Added a fix to now load layout without this race condition. This also fixes another bug, some communities don't load correctly, this is because of this race condition. To reproduce, try to open any community (lets say https://commonwealth-react.herokuapp.com/dydx/discussions) with Slow 3G network throttling enabled, this time the cow bobber will show infinitely.
2- New Proposal (`/new/proposal`) and View Proposal (`view/proposal`) pages updated to load chain without deferring.

## Test Plan 1 - For bug in original issue
- Visit the `/new/proposal` directly -> (Success: it should load correctly)
- Visit the `/new/proposal` page by first visiting a community and then selecting the `New on-chain proposal` option -> (Success: it should load correctly)

## Test Plan 2 - For bug with slow or unreliable connections
- Enable Slow 3G Network throttling in chrome devtools and visit a community discussions page (ex: https://commonwealth-react.herokuapp.com/dydx/discussions) -> (Success: it should load correctly)

## Deployment Plan
N/A

## Other Considerations
N/A
